src/model/orders/Order.java

```diff
@@ -0,0 +1,72 @@
+package model.orders;
+
+import model.Product;
+import model.persons.Client;
+import java.util.HashMap;
+import java.util.Map;
+
+import java.util.Collection;
+
+/**
+ * The Order Class has this Client and the list of OrderDetail (list of product and quantity)
+ */
+public class Order {
+    Integer id;
+    Client client;
+    Collection<OrderDetail> orderDetailList;
+    String note;
+
+    /**
+     * @return The price of the list of products
+     */
+    protected Double getProductListPrice(){
```

Comment:
Why is this protected? Why is this here? I think this should be outside of the class

Reply:
I had it private because I wanted the subclasses to call this methos instead of outside classes, but know that it's not abstract, I have to fix it